drivers/hmis/app/models/hmis/workflow_execution/instance.rb

```diff
@@ -6,6 +6,7 @@ module Hmis::WorkflowExecution
   class Instance < GrdaWarehouseBase
     belongs_to :template, class_name: 'Hmis::WorkflowDefinition::Template'
     has_many :steps, class_name: 'Hmis::WorkflowExecution::Step', dependent: :destroy
+    has_many :open_steps, -> { open.order_by_updated_at }, class_name: 'Hmis::WorkflowExecution::Step'
```

Comment:
Does this also need a TODO for 7647? It would be great if you can add a comment explaining why the sorting (ie if theres a particular schema field that depends on it)

Reply:
Whoops nevermind I see you have the todo on the scope itself

Reply:
There's no longer a schema field that grabs just the first one - I removed that in favor of returning them all and displaying them in a truncated list. I did add a comment though to try to clarify the purpose, thanks!